### PR TITLE
fix: video texture not reporting pause status after being paused

### LINKED ISSUE #1151: When pause videoTexture unity emits a videoEvent without pause status
When a video texture pause, it would be emit an event with Pause status, instead of that, it emits a event with Playing status, and it always changes currentOffset to 0.0 and videoLength to -1.

# Steps to reproduce

1. `git clone https://github.com/leamendoza97/video-streaming && cd video-streaming`
2. `git checkout feat/video-events && npm install`
3. Install the ECS PR with video event `npm install "https://sdk-team-cdn.decentraland.org/@dcl/js-sdk-toolchain/branch/feat/add-video-event-observable/decentraland-ecs-6.6.7-1211254954.commit-07752ef.tgz"`, any updates in https://github.com/decentraland/js-sdk-toolchain/pull/16
4.  Run the scene with `dcl build || true && dcl start` and open the dev tools to watch the console

Screenshot with console log
Screenshot_20210908_135838

### PR DESCRIPTION
**Pull request by Kinerius**
## What does this PR change?

Fix #1151 
WebVideoPlayer.jslib had its indentation fixed, the only change was line 170

## How to test

See #1151 repro steps

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by AjimenezDCL — APPROVED** [review]

**Comment by AjimenezDCL** [comment]
Looks good, I didnt check the jslib in details though

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/Plugins/WebVideoPlayer.jslib [hunk 1]
```diff
@@ -1,235 +1,236 @@
...
-
-  WebVideoPlayerSetLoop: function (videoId, value) {
-    videos[Pointer_stringify(videoId)].video.loop = value;
-  },
-
-  WebVideoPlayerSetPlaybackRate: function (videoId, value) {
-    videos[Pointer_stringify(videoId)].video.playbackRate = value;
-  },
+        videoData.video.pause();
+        videoData.state = 7; //PAUSED
```
Comment (pravusjif) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/Plugins/WebVideoPlayer.jslib:170]: can we use `videoData.state = videoState.PAUSED` here?

**Review by pravusjif — APPROVED** [review]
Looks good, left minor change suggestion.

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/Plugins/WebVideoPlayer.jslib (hunk 1 above)
Comment (Kinerius) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/Plugins/WebVideoPlayer.jslib:170]: videoState its not defined on that scope for some reason

**Pull request merged**